### PR TITLE
Add aspell recipes based on the ones from conda-recipes

### PR DESCRIPTION
**Pull request by asmeurer**
I replaced the manual binary prefix replacement with the automatic
detect_binary_files_with_prefix replacement.

### CODE REVIEW

File: recipes/aspell-en/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+./configure --vars ASPELL="$PREFIX/bin/aspell"
+make
+make install
```
Comment (ocefpaf) [inline recipes/aspell-en/build.sh]: Can we try `make check`?

File: recipes/aspell/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+./configure --prefix="$PREFIX"
+make
+make install
```
Comment (ocefpaf) [inline recipes/aspell/build.sh]: Same as above: `make check`

File: recipes/aspell/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package:
+  name: aspell
+  version: 0.60.7.20110707
+
+source:
+  fn: aspell-0.60.7-20110707.tar.gz
+  url: ftp://alpha.gnu.org/gnu/aspell/aspell-0.60.7-20110707.tar.gz
+  sha1: fcd3430a03fcda27d7359d6bbbc3c1396438c183
+  patches:
+    # http://www.freebsd.org/cgi/query-pr.cgi?pr=180565&cat=
```
Comment (ocefpaf) [inline recipes/aspell/meta.yaml]: Dead URL.

File: recipes/aspell/build.sh (hunk 2 above)
Comment (asmeurer) [inline recipes/aspell/build.sh]: I don't see any reference to `make check` in the aspell README.

Comment (ocefpaf) [inline recipes/aspell/build.sh]: If there is none don't worry. Thanks for checking!

**Comment by asmeurer** [comment]
What is going on with aspell-en. It has a configure file, and works for me.

**Comment by asmeurer** [comment]
I just remembered I don't actually use aspell, I use hunspell. So I've added that as well.

File: recipes/hunspell-en/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+package:
+  name: hunspell-en
+  version: 2014.02.10
```
Comment (asmeurer) [inline recipes/hunspell-en/meta.yaml]: I have no idea if this can be updated or what the story is there.

File: recipes/hunspell-en/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+source:
+  url: http://extensions.openoffice.org/en/download/17707
```
Comment (asmeurer) [inline recipes/hunspell-en/meta.yaml]: Apparently this always points to the latest version, which has already been updated.  Does anyone know how to navigate sourceforge? https://sourceforge.net/projects/aoo-extensions/

**Comment by asmeurer** [comment]
```
/bin/sh: aclocal-1.15: command not found
```

How do I fix this error (on Linux)?

**Comment by asmeurer** [comment]
I guess I'm going to have to actually figure out what to do with hunspell-en.

File: recipes/hunspell-en/meta.yaml (hunk 5 above)
Comment (patricksnape) [inline recipes/hunspell-en/meta.yaml]: https://sourceforge.net/projects/aoo-extensions/files/5266/21/access2base.oxt/download

**Comment by jakirkham** [comment]
Means you need `automake`.

File: recipes/hunspell-en/meta.yaml (hunk 5 above)
Comment (asmeurer) [inline recipes/hunspell-en/meta.yaml]: That doesn't seem to be the right file.

**Comment by asmeurer** [comment]
Didn't fix it. Maybe I need a specific version.

**Pull request closed without merging**